projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -11,7 +12,7 @@ import { TableMode } from '../table-api';
   providers: [SubscriptionLifecycle],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="table-controls">
+    <div class="table-controls" *ngIf="this.anyControlsEnabled">
```

Comment:
The changes in this file fix the issue where no controls are specified but extra padding is included